http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -66,6 +68,12 @@ case class ClusterCannotBeDeletedException(googleProject: GoogleProject, cluster
 case class ClusterCannotBeStartedException(googleProject: GoogleProject, clusterName: ClusterName, status: ClusterStatus)
   extends LeoException(s"Cluster ${googleProject.value}/${clusterName.value} cannot be started in ${status.toString} status", StatusCodes.Conflict)
 
+case class ClusterOutOfDateException()
+  extends LeoException(
+    "Your notebook runtime is out of date, and cannot be started due to recent updates in Terra. If you generated " +
+    "data or copied external files to the runtime that you want to keep please contact support. Otherwise, simply " +
```

Comment:
is there an email or link we can put in this message? just so user won't need to figure out where to go to contact support themselves

Reply:
Yes good call, will add a link.